### PR TITLE
Update preferences section in usermanual

### LINKED ISSUE #5811: documentation: new workflows
**Which new feature or section need to be documented/updated**
We need documenting the three workflows supported by dt:
- None
- Display referred
- Scene referred

### LINKED ISSUE #5815: documentation: font / dpi preferences
**Which new feature or section need to be documented/updated**
Font and DPI setting in preferences.

### LINKED ISSUE #5816: documentation: user's CSS snippet in prefs
**Which new feature or section need to be documented/updated**
The possibility to add CSS snipped directly from preferences should be documented.

### PR DESCRIPTION
**Pull request by Nilvus**
This update preferences section in usermanual to fit the new preferences dialog window.
Fix #5815 and fix #5816, thanks also to @elstoc first draft for general tab.

Still a WIP as many settings doesn't have descriptions. Will do it ASAP. Will also fix #5811 with next update.

### CODE REVIEW

**Comment by Nilvus** [comment]
Last commit add text for new workflows in preferences. Review requested by @aurelienpierre. Should be good as it inspired by 3.2 article and tooltip.

@TurboGit: ready to review.

File: doc/usermanual/lighttable/panels/selected_images.xml [hunk 1]
```diff
@@ -73,14 +73,15 @@
...
-        <quote>send files to trash when erasing images</quote> (<xref linkend="gui_options"/>)
+        <quote>send files to trash when erasing images</quote> (<xref linkend="security"/>)
         this either irrevocably deletes the images from filesystem or puts them into your
         system's trash bin.
       </para>
 
       <para>
         See also preference option <quote>ask before erasing images from disk</quote>
-        (<xref linkend="gui_options"/>). If this configuration option is not active, darktable
+        (<xref linkend="security
```
Comment (TurboGit) [inline doc/usermanual/lighttable/panels/selected_images.xml:83]: I fear this breaks the link because of new-line in string.

**Review by TurboGit — CHANGES_REQUESTED** [review]
A minor change needed. Big work ! Thanks.

File: doc/usermanual/lighttable/panels/selected_images.xml (hunk 1 above)
Comment (Nilvus) [reply doc/usermanual/lighttable/panels/selected_images.xml]: And a not wanted one I missed. Fixed.

**Review by matt-maguire — CHANGES_REQUESTED** [review]
After the paragraph:
`This workflow consist of adding, by default, +0/5 EV of exposure (in exposure module), to adjust the global brightness and mid-grey to a similar value to that used by most JPEGs produced by SLR cameras.`
I would suggest to add the remarks:

`This default +0/5 EV value can be overridden using an auto-apply preset if it is found the images from a particular camera are consistently too dark.`

`The scene-referred workflow also enables by default the "compensate camera exposure" option in the exposure module, to adjust the global brightness appropriately in cases where the camera's exposure compensation dial was used to protect highlights in the image or Expose To The Right (ETTR) to optimally make use of the sensor's dynamic range.`

**Comment by TurboGit** [comment]
@Nilvus : I think @matt-maguire proposal is good, can you have a look? TIA.

**Comment by Nilvus** [comment]
@matt-maguire and @TurboGit: done. Ready to merge!

**Comment by TurboGit** [comment]
Great! Thanks, merging now.

**Pull request merged**